internal/services/cdn/cdn_frontdoor_origin_resource.go

```diff
@@ -8,8 +8,11 @@ import (
 	"time"
 
 	"github.com/Azure/azure-sdk-for-go/services/cdn/mgmt/2021-06-01/cdn" // nolint: staticcheck
```

Comment:
Just to check, are we leaving some resources out of this for scope reasons, or technical issues?

Reply:
Correct, I am upgrading the resources as feature requests come in from internal/external customers.